demisto_sdk/commands/setup_env/setup_environment.py

```
@@ -0,0 +1,406 @@
+            f"Failed to get secret {secret_id} from secret manager, skipping"
+        )
+        return {}
+    # Return the decoded payload.
+    payload = response.payload.data.decode("UTF-8")
+    return json5.loads(payload).get("params")
+
+
+def add_init_file_in_test_data(integration_script: IntegrationScript):
+    if (integration_script.path.parent / "test_data").exists():
+        (integration_script.path.parent / "test_data" / "__init__.py").touch()
+
+
+def configure_dotenv():
+    """
+    This functions configures the .env file located with PYTHONPATH and MYPYPATH
+    """
+    dotenv_path = CONTENT_PATH / ".env"
+    env_vars = dotenv.dotenv_values(dotenv_path)
+    env_vars["PYTHONPATH"] = ":".join([str(path) for path in PYTHONPATH])
+    env_vars["MYPYPATH"] = ":".join([str(path) for path in PYTHONPATH])
+    for key, value in env_vars.items():
+        if not value:
+            continue
+        dotenv.set_key(dotenv_path, key, value)
+
+
+def configure_vscode_settings(
+    ide_folder: Path, integration_script: IntegrationScript, interpreter_path: Path
+):
+    shutil.copy(Path(__file__).parent / "settings.json", ide_folder / "settings.json")
+    with open(ide_folder / "settings.json") as f:
+        settings = json5.load(f)
+
+    settings["python.defaultInterpreterPath"] = str(interpreter_path)
+    settings["python.testing.cwd"] = str(integration_script.path.parent)
+    with open(ide_folder / "settings.json", "w") as f:
+        json.dump(settings, f, indent=4)
+
+
+def configure_vscode_tasks(
+    ide_folder: Path, integration_script: IntegrationScript, test_docker_image: str
+):
+    if integration_script.type == "powershell":
+        logger.debug("Powershell integration, skipping tasks.json")
+        return
+    docker_python_path = []
+    for path in PYTHONPATH:
+        with contextlib.suppress(ValueError):
+            # we can't add paths which is not relative to CONTENT_PATH, and `is_relative_to is not working on python3.8`
```

can we have a better test for this specific error? Currently, any `ValueError` will be ignored, could be dangerous

---

the only thing we do here is `relative_to`. The only failure for this method will be when path a is not a relative of path b, and we want to ignore in this case.
Maybe we should raise an error if the PYTHONPATH is empty in the end (or doesn't contain CommonServerPython)